src/source/src/connector_source.rs

```diff
@@ -179,6 +184,32 @@ impl ConnectorSourceReader {
                         tracing::warn!("message parsing failed {}, skipping", e.to_string());
                         continue;
                     }
+
+                    let new_op_num = builder.op_num();
+
+                    // new_op_num - old_op_num is the number of rows added to the builder
+                    for _ in old_op_num..new_op_num {
```

Comment:
There is a problem with this. What if `_rw_kafka_timestamp` is not in last column? For example following sql:
```select a, _rw_kafka_timestamp, b from kafka```?

I think we can pass `SourceMeta` to row writer and process meta column in `do_action` method.